lib/gemfury/command/app.rb

```diff
@@ -221,12 +224,22 @@ def push_files(command, gem_paths)
       die!("Problem: No valid packages found", nil, command)
     end
 
+    push_options = { }
+    if options[:public]
+      push_options[:public] = true
+    end
+
     # Let's get uploading
     files.each do |file|
       begin
         shell.say "Uploading #{File.basename(file.path)} "
-        client.push_gem(file)
-        shell.say "- done"
+        client.push_gem(file, push_options)
+
+        if options[:public]
```

Comment:
I'm not a huge fan of this implementation here, particularly basing output on the CLI flag. Imagine this particular case:

```sh
fury push --public pkg-0.0.2.gem  # This will output "privacy: public"
fury push pkg-0.0.3.gem           # This will not
```

Although both 0.0.2 and 0.0.3 are now public, only the first one prints this output, which is inconsistent.

If we want to output package privacy, we should do it based on what the `client.push_gem` returns (`private` field for the Version's parent Package).

If the API output doesn't return the `private` field at this time, let's scrap the extra output for now and implement it as a separate feature.